tests/features/support/Timeouts.js

```diff
@@ -0,0 +1,5 @@
+var {defineSupportCode} = require('cucumber');
+
+defineSupportCode(function({setDefaultTimeout}) {

```

Comment:
__[jshint]__
* 'destructuring binding' is available in ES6 (use 'esversion: 6') or Mozilla JS extensions (use moz).

<!-- Comment by [SideCI](https://sideci.com) -->